src/amo/components/Collection/index.js

```diff
@@ -248,6 +255,7 @@ export class CollectionBase extends React.Component<Props> {
             addons={addons}
             editing={editing}
             loading={!collection || loading}
+            placeholderText={placeholderText}
```

Comment:
Rather than pass a `placeholderText` to the `AddonsCard`, I think it would make more sense to just display the placeholder text here *instead* of an `AddonsCard`. If you want to encapsulate the logic/formatting of the placeholder text in a single place, I'd create a new component for it in `src/ui/components/`, but that might be overkill.